packages/rum/src/domain/record/observer.ts

```diff
@@ -88,6 +112,7 @@ export function initObservers(o: ObserverParam): ListenerHandler {
   const styleSheetObserver = initStyleSheetObserver(o.styleSheetRuleCb)
   const focusHandler = initFocusObserver(o.focusCb)
   const visualViewportResizeHandler = initVisualViewportResizeObserver(o.visualViewportResizeCb)
+  const frustartionHandler = initFrustrationObserver(o.lifeCycle, o.frustrationCb)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** typo‏
```suggestion
  const frustrationHandler = initFrustrationObserver(o.lifeCycle, o.frustrationCb)
```